go.mod

```diff
@@ -70,13 +70,12 @@ require (
 )
 
 replace (
-	github.com/golang/protobuf => github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf => github.com/golang/protobuf v1.5.1
 	github.com/opencontainers/runtime-spec => github.com/opencontainers/runtime-spec v1.0.3-0.20201121164853-7413a7f753e1
 	// Pinning the syndtr/gocapability until https://github.com/opencontainers/runc/commit/6dfbe9b80707b1ca188255e8def15263348e0f9a
 	// is included in the runc release
 	github.com/syndtr/gocapability => github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2
-	google.golang.org/genproto => google.golang.org/genproto v0.0.0-20200117163144-32f20d992d24
```

Comment:
@saschagrunert, okay, seems that I got the main difference that's causing the errors we're facing in the CI.
Some time ago when I tested this I didn't remove genproto, and on local tests seems that just by keeping genproto pinned is enough to have kata-containers happy, even if we remove the pinned protobuf and grpc.

Do you think that leaving genproto pinned could cause any issue?  If no, please, just update the PR to remove the pin on both protobuf & grpc, leaving the genproto pinned, and we'd be good to go.

If it cause some issue on the CRI-O side, what's the issue caused?

Please, see https://github.com/cri-o/cri-o/pull/4957, which used a still pinned version but didn't remove genproto. (CI: http://jenkins.katacontainers.io/job/kata-containers-2-crio-PR/168/console)